samcli/commands/sync/command.py

```diff
@@ -284,7 +284,7 @@ def do_cli(
             return
         global_config.set_accelerate_opt_in_stack(template_file, stack_name)
     else:
-        LOG.info(Colored().yellow(SYNC_INFO_TEXT))
+        LOG.info(Colored().color_log(msg=SYNC_INFO_TEXT, color="yellow"), extra=dict(markup=True))
```

Comment:
nit and not blocking: Let's think about creating some constants that might make it a little easier to keep consistency and allow us to define the colors to use for each scenario. Something like having

```
Colors.Success = "green"
Colors.Warning = "yellow"
Colors.Message = "whatever we decide"
...
```